### PR TITLE
handle save_y2logs when content modified when creating tarball

### PR DESCRIPTION
**Pull request by jreidinger**
How it looks now ( easy to reproduce with Y2DEBUG=1 and slide show ):
save_y2logs_modified

### CODE REVIEW

File: scripts/save_y2logs [hunk 1]
```diff
@@ -142,11 +142,14 @@ echo "Saving YaST logs to $TARGET" >&2
...
 [ -n "$COMPRESSION" ] && COMPRESSION="--$COMPRESSION"
 
-tar cf "$TARGET" $COMPRESSION --dereference -C / $LIST -C /var/log $LIST_VAR -C $TMPDIR $LIST_TMP
+tar cf "$TARGET" --warning=no-file-changed $COMPRESSION --dereference -C / $LIST -C /var/log $LIST_VAR -C $TMPDIR $LIST_TMP
 err=$?
 
 rm -rf $TMPDIR
 
+# tar returns 1 when content of tarball is older then current system. Happen when something still writting to logs or memsampler (bsc#1182710#c2)
+[ $err == 1 ] && echo "Warning: logs modified during save. Yast is probably running. Content of tarball can be older then current content." >&2 && err=0
```
Comment (shundhammer) [inline scripts/save_y2logs:151]: older **than**

File: scripts/save_y2logs [hunk 2]
```diff
@@ -142,11 +142,14 @@ echo "Saving YaST logs to $TARGET" >&2
 
 [ -n "$COMPRESSION" ] && COMPRESSION="--$COMPRESSION"
 
-tar cf "$TARGET" $COMPRESSION --dereference -C / $LIST -C /var/log $LIST_VAR -C $TMPDIR $LIST_TMP
+tar cf "$TARGET" --warning=no-file-changed $COMPRESSION --dereference -C / $LIST -C /var/log $LIST_VAR -C $TMPDIR $LIST_TMP
 err=$?
 
 rm -rf $TMPDIR
 
+# tar returns 1 when content of tarball is older then current system. Happen when something still writting to logs or memsampler (bsc#1182710#c2)
```
Comment (shundhammer) [inline scripts/save_y2logs:150]: NP: older **than the** current system

File: scripts/save_y2logs (hunk 1 above)
Comment (jreidinger) [reply scripts/save_y2logs:151]: thanks, will fix

**Review by shundhammer — APPROVED** [review]
LGTM

**Pull request merged**